source/install/verify-signatures/gpg.txt

```diff
@@ -0,0 +1,93 @@
+.. _mongosh-verify-signatures-gpg:
+
+==========================================
+Verify Packages with GPG (Linux and macOS)
+==========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. include:: /includes/verify-signatures-intro.rst
+
+This page describes how to use GPG to verify Linux and macOS packages.
+
+Before you Begin
+----------------
+
+.. include:: /includes/verify-signatures-before-you-begin.rst
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Import the MongoDB Shell public key
+
+      Run this command:
```

Comment:
(optional minimalism)
I think you could omit the intro sentence "Run this command:" That should be covered by the step title.